azerothcore/azerothcore-wotlk
# fix(DB/Quest) Class Training quest for Sunstrider Isle

## Changes Proposed:
-  Add required completed quest to quest addon

## Issues Addressed:
- Training quest should not be available until after Quest 8325 Reclaiming Sunstrider Isle is complete.

## SOURCE:
Checked several classes on official. These quest are not available until Reclaiming Sunstrider Isle is complete.

## Tests Performed:
Tested works properly


## How to Test the Changes:
Make a blood elf toon and notice training quest is available before Reclaiming Sunstrider Isle is complete. Toon cannot train until level 2.
